src/orca/services/synapse/ops.py

```diff
@@ -1,8 +1,9 @@
 import logging
 from dataclasses import field
 from functools import cached_property
-from typing import ClassVar
+from typing import ClassVar, List, Union
 
+from challengeutils import utils
```

Comment:
This package is going to be deprecated in favor of cnb-tools eventually. For now it's ok, but the package isn't being regularly maintained.

Probably need to add this to the pipfile.